### DISCUSSION TITLE
Can't figure out how to transfer money from daily account to credit card

### DISCUSSION THREAD

**Discussion by ipsi** [firefly-iii/firefly-iii#7889] [Firefly III] [answered]
This seems like it should be really simple, but I can't figure out how to do it. I have a credit card asset account (it has an account role of `Credit Card`). It has a negative balance of, say, €1,000, as various expenses have it as their source account. On a regular basis, the credit card company takes the balance out of my daily account as a direct debit - normal practice.

I would have expected to be able to make a transfer between the two accounts, but that's not possible as I can't select the credit card as a destination account. This also means I can't have a "revenue" account sending money to my credit card, and receiving money from my daily account. Transaction amounts can't be negative, so I can't be cheeky and set my credit card as a source account.

Checking the page source, I can see a bunch of references to the various account types (e.g., the variable `allowedOpposingTypes`), and credit card is completely omitted - not sure if this intentional or an oversight?

If this description isn't sufficiently clear, I'm happy to upload screenshots or a short video (<30s) showing what I mean.

EDIT:
**Version**: 6.0.20
**Deployment**: Firefly & Data Importer, Docker Compose

**Comment by JC5** [comment]
I can't replicate this on the demo site. The auto complete comes back with the credit card as expected.

**Reply by ipsi** [reply, marked as answer]
I gave it another shot, and it's working now - I'd swear that last night it was refusing to display the credit card in the destination, even if I tried typing in the name. No idea what changed, but not going to worry unless it happens again.
